src/lib/chains/evm/accounts.ts

```diff
@@ -0,0 +1,62 @@
+import * as ethers from "ethers"   ;
+import { BridgeAccounts, BridgeAccountManager } from "../../common/interfaces/interfaces";
+import { AlgorandAccount } from "../algorand";
+import { SolanaAccount } from "../solana";
+export type EvmAccount = {
+    addr:string, 
+    pk:string,
+    mnemonic:string
+}
+
+export class EvmAccounts implements BridgeAccountManager<BridgeAccounts> {
+
+    private _accounts: Record<string, EvmAccount> = {};
+    public async add( ...args: [sk: Uint8Array | undefined] | [mnemonic: string | undefined] ):Promise<EvmAccount> {
+    const mnemonic_ = args[0] as string;
+    const wallet = await  ethers.Wallet.fromMnemonic(mnemonic_)
+    const privateKey = wallet.privateKey;
+    const address = wallet.address;
+    const evmAccount:EvmAccount = {
+        addr:address,
+        pk:privateKey,
+        mnemonic:mnemonic_
+    }
+    this._accounts[address] = evmAccount; 
+    return Promise.resolve(evmAccount)
+    }
+
+    public async createNew():Promise<EvmAccount>{
```

Comment:
why is there a `createNew` method? end user would like be using their mnemonic or private key